test/Main.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module Main (main) where
+
+import Data.Hashable (Hashable)
+import Data.Proxy (Proxy (Proxy))
+import Data.Text (Text)
+import GHC.Generics (Generic)
+import Test.QuickCheck (Arbitrary, arbitrary, choose, vectorOf)
+
+import qualified Data.Text as T
+import qualified Map.Laws.Hash as Hash
+import qualified Map.Laws.Int as Int
+import qualified Map.Laws.Ord as Ord
+
+
+-- | ShortText is a Text with length [0, 3] chosen "arbitrarily"
+newtype ShortText = ShortText Text
```

Comment:
Should this be named field `unShortText`?

Reply:
@vrom911 Yes, I guess it should be made according to our style guide. This code was written a long time ago...